install/devsds/lib/lvm.sh

```diff
@@ -194,10 +253,21 @@ osds::lvm::clean_backing_file() {
     fi
 }
 
+osds::lvm::clean_volume_group_for_file() {
+    local fvg="opensds-files-default"
+    osds::lvm::remove_volume_group_for_file $fvg
+    # if there is no logical volume left, it's safe to attempt a cleanup
+    # of the backing file
+    if [[ -z "$(sudo lvs --noheadings -o lv_name $fvg 2>/dev/null)" ]]; then
```

Comment:
wondering why we are using lvs somewhere and lvm most places? can't we use lvm everywhere. Also, command absolute path should be used.